strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/TrustStoreServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.carlspring.strongbox.services.impl;
+
+import org.carlspring.strongbox.security.certificates.KeyStoreManager;
+import org.carlspring.strongbox.services.TrustStoreService;
+import org.carlspring.strongbox.services.support.TrustStoreCertificationAdditionException;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.URL;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Service
+public class TrustStoreServiceImpl
+        implements TrustStoreService
+{
+
+    private static final String PASSWORD = "password";
+
+    @Value("classpath:etc/ssl/truststore.jks")
```

Comment:
Will this work when the standalone distribution is started? We'll have to check whether this directory is included in the classpath.

Reply:
Yeah, directory is included in the classpath.

However, there will be an error once trying to put something in this file since it does not reside in the file system but is packaged in the jar file. Good point! :)